build.sbt

```
@@ -73,7 +73,8 @@ lazy val scalacOptions_2_13 = Seq(
   "-Xfatal-warnings",
   "-Ywarn-dead-code",
   "-Ymacro-annotations",
-  "-Xlint:-byname-implicit"
+  "-Xlint:-byname-implicit",
+  "-Xlint:strict-unsealed-patmat"
```

Hmm we have `-Xlint` on 2.12 but not on 2.13
Let's enable everything and see what breaks?
```suggestion
  "-Xlint:_,-byname-implicit"
```